FSQLocationBrokerTests/FSQTestingHelper.h

```diff
@@ -0,0 +1,47 @@
+//
+//  FSQTestingHelper.h
+//  FSQLocationBroker
+//
+//  Created by Eric Bueno on 4/18/17.
+//  Copyright © 2017 Foursquare. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import <CoreLocation/CoreLocation.h>
+
+#pragma mark Date Equality Macros
+
+#define FSQAssertEqualDates(date1, date2) \
```

Comment:
I was thinking of just a C function. If XCTAssert is implemented as a macro, I'm ok with keeping it that way. I'm a fan of using C functions for things like this though - it doesn't need to belong to a class or object, and matches the style of the built-in XCTAssert macro...